queries/period_slippage.sql

```diff
@@ -221,6 +221,13 @@ internal_buffer_trader_solvers as (
     -- Exclude services and test solvers
     and environment not in ('service', 'test')
 ),
+-- PoC Query For Token List: https://dune.com/queries/1547103
```

Comment:
FYI: Default parameter is still with 0x here.

Reply:
Yes. That has always been the case. Is there some reason you think this should be different?

Reply:
In my opinion you're concatenating the 0x now, therefore I thought it shouldn't be there. Hence, the way you have it now it won't run for me. 

Here's my [forked version](https://dune.com/queries/1553380) with how I think it should be. Please let me know if I'm missing something.

Reply:
:+1: